src/App.css

```diff
@@ -1,33 +1,3 @@
-@font-face {
-  font-family: 'Anton';
-  font-style: normal;
-  font-weight: normal;
-  font-display: swap;
-  src: local('Anton Regular') local('Anton-Regular'),
-       url('./assets/fonts/anton-v15-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
-       url('./assets/fonts/anton-v15-latin-regular.woff') format('woff'), /* Modern Browsers */
-}
-
-@font-face {
-  font-family: 'Codystar';
-  font-style: normal;
-  font-weight: 400;
-  font-display: swap;
-  src: local('Codystar Regular') local('Codystar-Regular'),
-       url('./assets/fonts/codystar-v10-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
-       url('./assets/fonts/codystar-v10-latin-regular.woff') format('woff'), /* Modern Browsers */
-}
-
-@font-face {
-  font-family: 'Oswald';
-  font-style: normal;
-  font-weight: 400;
-  font-display: swap;
-  src: local(''),
-       url('./assets/fonts/oswald-v40-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
-       url('./assets//fonts/oswald-v40-latin-regular.woff') format('woff'), /* Modern Browsers */
-}
-
```

Comment:
You were using both Google Fonts _and_ loading custom font-files. Obviously you only need one of the two. I chose to remove the custom font-files since Google Fonts takes away some of the headache of having your own custom font-files in your repo, but either is fine.